pyccel/commands/epyccel.py

```diff
@@ -359,6 +359,8 @@ def epyccel( python_function_or_module, **kwargs ):
                 fun_name = python_function_or_module.__name__ if fun else None
                 success  = True
             # error handling carried out after broadcast to prevent deadlocks
+            except  PyccelError as e:
```

Comment:
Excessive whitespace?
```suggestion
            except PyccelError as e:
```

Reply:
:facepalm: copy-paste typo :|